console_conf/ui/views/chooser.py

```diff
@@ -47,23 +47,36 @@
 log = logging.getLogger("console_conf.views.chooser")
 
 
-class ChooserCurrentSystemView(BaseView):
+class ChooserBaseView(BaseView):
     title = "Ubuntu Core"
 
-    def __init__(self, controller, current, has_more=False):
-        fmt = "Select one of available actions for \"{}\" by \"{}\"{}."
-        maybe_more = " or view all available systems" if has_more else ""
-        excerpt = fmt.format(current.model.display_name,
-                             current.brand.display_name,
-                             maybe_more)
+    def __init__(self, current, scr):
+        if current is not None:
```

Comment:
Same question, could we call `__init__()` as the first thing and only then assign to `self.title`? I find this style more in line with what ought to happen in a perfect scenario.